pkg/vms/hostvmslist.es6

```diff
@@ -0,0 +1,102 @@
+/*jshint esversion: 6 */
+import React, { PropTypes } from "react";
+import { shutdownVm, forceVmOff, forceRebootVm, rebootVm, startVm, navigate, setRefreshInterval } from "./actions.es6";
+import { ReloadSwitch } from "./reloadSwitch.es6";
+import { VMS_CONFIG } from "./config.es6";
+import { StateIcon } from "./vmdetail.es6";
+import {htmlA, htmlDiv, htmlH2, htmlI} from "./helpers.es6";
+
+const { object, func } = PropTypes;
+
+function NoVm () {
+  return htmlDiv("cockpit-log-warning",
+      htmlDiv("blank-slate-pf",
+        htmlDiv("blank-slate-pf-icon",
+          htmlI("fa fa-rocket")),
+        React.createElement("span", null,
+          React.createElement("h1", null, "No VM is running on this host"),
+          React.createElement("p", null, "Let's keep hoping in a change or start a new one ..."))
+      ));
+}
+
+// TODO: Add actual usage
+function VmUsageCharts ({vm}) {
```

Comment:
Lets remove this code for this pull request and add it later when this part of the feature is ready.